api/graphite.go

```diff
@@ -207,6 +207,12 @@ func (s *Server) renderMetrics(ctx *middleware.Context, request models.GraphiteR
 
 	exprs, err := expr.ParseMany(request.Targets)
 	if err != nil {
+		err := response.WrapError(err)
+		if err.Code() == http.StatusBadRequest && !request.NoProxy {
+			log.Infof("Proxying to Graphite because of error: %s", err.Error())
```

Comment:
can we add the request TraceId, so that we can correlate this log message with the final log message for the proxied request.  This will allow us to check if the request was successful in graphite or not.

Reply:
see https://github.com/grafana/metrictank/pull/1695 for the start of updating all request logs to include the TraceID